res/css/views/rooms/_EventTile.scss

```diff
@@ -45,14 +45,14 @@ limitations under the License.
 .mx_EventTile .mx_SenderProfile {
     color: $primary-fg-color;
     font-size: 14px;
-    display: block; /* anti-zalgo, with overflow hidden */
+    display: inline-block; /* anti-zalgo, with overflow hidden */
     overflow-y: hidden;
     cursor: pointer;
     padding-left: 65px; /* left gutter */
     padding-bottom: 0px;
     padding-top: 0px;
     margin: 0px;
-    line-height: 22px;
+    line-height: 17px;
```

Comment:
What's the reason for changing this value? (I guess I am not sure why either 22 or 17 is a good value here, but anyway it seems unrelated to the main issue you want to fix.)

Reply:
Visually, Firefox and Chrome don't have quite the same line-height right now because it looks like FF is not using that field anyway in this case. Changing 22px to 17px makes them look visually the same *after* changing the display value.

Reply:
Okay, thanks for the detail. Seems reasonable to me then.